src/components/ApplicationTable/ApplicationTable.js

```diff
@@ -3,11 +3,48 @@ import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import styled from 'styled-components';
 import { rem } from 'polished';
-import { SPACE } from 'config';
-import Table, { TableWrapper } from 'components/Table';
+import media from 'utils/media';
+import { SPACE, COLORS } from 'config';
 import Text from 'components/Text';
 import * as icons from './icons';
 
+const TableWrapper = styled(Text).attrs({
+  as: 'div',
+  color: 'black',
+  fontSize: [1, 2],
+  heavy: true,
+}) `
+  overflow-x: auto;
+  margin-right: -${rem(SPACE[4])};
+  padding-right: ${rem(SPACE[4])};
+  margin-left: -${rem(SPACE[4])};
+  padding-left: ${rem(SPACE[4])};
+  margin-bottom: 100px;
```

Comment:
please make sure, that padding and margin values can not be added via props, for example `mx={-4}`, `px={4}` etc

Reply:
Fixed